src/components/common.js

```diff
@@ -92,8 +92,8 @@ export const Clickable = forwardRefWithName('Clickable',
 export const Link = forwardRefWithName('Link', ({ disabled, variant, children, baseColor = colors.accent, ...props }, ref) => {
   return h(Clickable, _.merge({
     ref,
-    style: { // 0.72 is the min to meet ANDI's contrast requirement
-      color: disabled ? colors.dark(0.72) : baseColor(variant === 'light' ? 0.3 : 1),
+    style: { // Note: 0.72 is the min to meet ANDI's contrast requirement
```

Comment:
It looks like this comment is no longer relevant since the code with 0.72 has been removed. Since this PR is because the contrast for disabled elements is too low, was this comment ever accurate?

Reply:
IIRC at the time that comment was added, the related element was passing ANDI's checks.

Reply:
@nawatts disabled items don't have to meet color contrast standards. https://www.w3.org/TR/WCAG21/#contrast-minimum The disabled item is an "inactive user interface component".

Reply:
Today I learned. That makes things easier.

Reply:
I have learned, forgotten, and relearned this several times. :)